MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: post
+title: Validation in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the validation feature in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Validation in .NET MAUI DataForm (SfDataForm)
+
+The data form validates the data and displays hints in the case of validation is not passed. In case of invalid data, the error message is shown at the bottom of the editor.
```

Comment:
The data form validates the data and user input in order to update the correct value in the underlying data object. In case of invalid data, the error message is shown at the bottom of the editor.

Reply:
content changed